Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
+configuration:
+- display: Demisto Server URL
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Demisto Server API Key
+  name: apikey
```

Comment:
I would not enforce API Key, it should support user/pass as well, easy to implement.

Reply:
disagree here. we can discuss when I'm back.

Reply:
Most of the users don't remember their api key or don't have access to get it (only admin can).
You can disagree, but the correct way it so support both user/pass and api key.

Reply:
Again - disagree :)
integrating systems in most apps will be done by API key. Passwords may expire (if we add this).